packages/fast-components-styles-msft/src/pivot/index.ts

```diff
@@ -0,0 +1,106 @@
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import {
+    applyFocusVisible,
+    applyLocalizedProperty,
+    Direction,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { ensureBrandNormal, hoverContrast } from "../utilities/colors";
+import outlinePattern from "../patterns/outline";
+import { PivotClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+const styles: ComponentStyles<PivotClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<PivotClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        pivot: {
+            position: "relative",
+            color: designSystem.foregroundColor,
+            overflow: "hidden",
+        },
+        pivot_tabList: {
+            display: "flex",
+            boxSizing: "border-box",
+        },
+        pivot_tab: {
+            minHeight: "32px",
+            padding: "0 12px",
+            fontSize: "14px",
+            whiteSpace: "nowrap",
+            display: "flex",
+            border: "2px solid",
+            borderColor: "transparent",
+            alignItems: "center",
+            boxSizing: "border-box",
+            userSelect: "none",
+            borderRadius: toPx(designSystem.cornerRadius),
+            "&:hover": {
+                opacity: "0.8",
+            },
+            "&:active": {
+                opacity: "0.6",
+            },
+            ...applyFocusVisible({
+                borderColor: designSystem.foregroundColor,
+            }),
+        },
+        pivot_tab__active: {},
+        pivot_activeIndicator: {
+            position: "absolute",
+            borderRadius: toPx(designSystem.cornerRadius),
+            top: "27px",
+            left: "-10px",
+            transition: "0.2s ease-in-out",
+            width: "20px",
+            height: "3px",
+            display: "block",
+            background: ensureBrandNormal,
+        },
+        pivot_tabPanel: {
+            display: "block",
+        },
+        pivot_tabPanel__hidden: {
+            display: "none",
+        },
+        pivot_tabPanels: {
+            animationTimingFunction: "cubic-bezier(0.4, 0.0, 0.6, 1.0)",
+        },
+        pivot_tabPanels__fromLeft: {
+            animation: "fromLeft 0.2s",

```

Comment:
We tend to use `previous` and `next` instead of `left` and `right`, as to not confuse in LTR/RTL

Reply:
I think we should rename all

Reply:
Agree on the classNames animation names I will keep as is and swap those based on direction